# HG changeset patch
# User Byoung-Gi Lee <handrake@gmail.com>
# Date 1332924570 -32400
# Node ID bb5673526b145ab3f8331a2b411cef6f502ebc28
# Parent  6346d4a7e7210bb0d8829be6e97db9d644ea4355
produced should point to an industry's last month production.

diff -r 6346d4a7e721 -r bb5673526b14 src/station_cmd.cpp
--- a/src/station_cmd.cpp	Sun Mar 25 23:42:49 2012 +0000
+++ b/src/station_cmd.cpp	Wed Mar 28 17:49:30 2012 +0900
@@ -490,7 +490,7 @@
 
 		for (uint j = 0; j < lengthof(i->produced_cargo); j++) {
 			CargoID cargo = i->produced_cargo[j];
-			if (cargo != CT_INVALID) produced[cargo]++;
+			if (cargo != CT_INVALID) produced[cargo] += i->last_month_production[j];
 		}
 	}
 
